### PR TITLE
[DellEMC] S6100 -  Adding logger to fetch SSD FW Upgrade status

### PR DESCRIPTION
**Pull request by santhosh-kt**
#### Why I did it
- To log SSD FW upgrade attempts in syslog.

#### How I did it
- Adding logger.

#### Which release branch to backport (provide reason below if selected)

- [x] 201811
- [x] 201911
- [ ] 202006
- [x] 202012
- [ ] 202106
- [x] 202111
- [x] 202205
- [x] 202211

### CODE REVIEW

**Comment by vdahiya12** [comment]
LGTM

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh [hunk 1]
```diff
@@ -36,6 +36,7 @@ if [ -e $SSD_FW_UPGRADE/GPIO7_pending_upgrade ]; then
 fi
 
 echo "$0 `date` SSD FW upgrade logs post reboot." >> $SSD_UPGRADE_LOG
+logger -p user.crit -t DELL_S6100_SSD_MON "SSD FW upgrade logs post reboot."
```
Comment (prgeor) [inline platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh:39]: can you add syslog at line 29 and line 33 as well before `exit 0`

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh [hunk 2]
```diff
@@ -71,11 +72,14 @@ elif [ $SSD_FW_VERSION == "s210506g" ] || [ $SSD_FW_VERSION == "s16425cg" ]; the
         rm -rf $SSD_FW_UPGRADE/GPIO7_*
         touch $SSD_FW_UPGRADE/GPIO7_high
         systemctl start --no-block s6100-ssd-monitor.timer
+        logger -p user.crit -t DELL_S6100_SSD_MON "SSD FW upgraded already."
```
Comment (prgeor) [inline platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh:75]: Please add a syslog to print ssd STATUS 1 and 2 at line 43, 46

Comment (prgeor) [inline platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh:75]: Please print the model and SSD version in the log

**Comment by santhosh-kt** [comment]
Updated the changes based on the comments.

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh [hunk 3]
```diff
@@ -93,6 +103,7 @@ else
         touch $SSD_FW_UPGRADE/GPIO7_pending_upgrade
```
Comment (prgeor) [inline platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh:103]: what if the SSD fw is OLD and still 101 line returns False,  shouldn't we ` touch $SSD_FW_UPGRADE/GPIO7_pending_upgrade` in that case?

**Review by prgeor — APPROVED** [review]

File: platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh (hunk 3 above)
Comment (santhosh-kt) [reply platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh:103]: SSD_UPGRADE_STATUS1 and SSD_UPGRADE_STATUS2 are ff in default.
Hence, it wont get False incase no SSD attempt is attempted

**Pull request merged**